crates/bevy_ecs/src/system/system.rs

```diff
@@ -31,7 +31,7 @@ impl SystemId {
 ///
 /// Systems are executed in parallel, in opportunistic order; data access is managed automatically.
 /// It's possible to specify explicit execution order between specific systems,
-/// see [SystemDescriptor](crate::schedule::SystemDescriptor).
+/// see [SystemConfig](super::SystemConfig) and the other files in the [./config](super::config) directory.
```

Comment:
```suggestion
/// see [SystemConfig](super::SystemConfig) and the other members of the [`config` module](super::config).
```

Users won't / shouldn't have to dig through the source, so "directory" is inappropriate.